### PR TITLE
Fix update_task: Download zip to sufficiently large space.

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: src/clusterfuzz/_internal/bot/tasks/update_task.py [hunk 1]
```diff
@@ -194,8 +194,7 @@ def update_source_code():
   shell.clear_temp_directory()
 
   root_directory = environment.get_value('ROOT_DIR')
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/update_task.py:196]: Can we use the parent dir instead? Might be some weird issues with downloading the zip into the directory we're unzipping to.

**Review by oliverchang — APPROVED** [review]

File: src/clusterfuzz/_internal/bot/tasks/update_task.py [hunk 2]
```diff
@@ -270,6 +270,7 @@ def update_source_code():
                                      utils.LOCAL_SOURCE_MANIFEST)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/update_task.py:270]: Do we need to udpate this too?

File: src/clusterfuzz/_internal/bot/tasks/update_task.py [hunk 3]
```diff
@@ -193,9 +193,10 @@ def update_source_code():
   process_handler.cleanup_stale_processes()
   shell.clear_temp_directory()
 
-  root_directory = environment.get_value('ROOT_DIR')
-  temp_directory = environment.get_value('BOT_TMPDIR')
-  temp_archive = os.path.join(temp_directory, 'clusterfuzz-source.zip')
+  # ROOT_DIR just means the clusterfuzz directory.
+  source_root_directory = environment.get_value('ROOT_DIR')
+  root_directory = os.path.dirname(source_root_directory)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/update_task.py:198]: Re-using the root_Directory variable here might be confusing. Can we call this something else?

**Review by oliverchang — CHANGES_REQUESTED** [review]

**Comment by jonathanmetzman** [comment]
Is new name good or shoudl I go back to "output_directory"

**Review by oliverchang — APPROVED** [review]

**Pull request merged**